wiki/en/Software-Manual.md

```diff
@@ -28,7 +28,7 @@ This manual documents the Jamulus client application for use by musicians and si
 
 ## Ping, Delay and Jitter
 
-**Ping** shows your network latency in milliseconds, the lower the better. Ping time contributes to overall delay (see below). The most probable causes for a high ping are that your distance to the server is too large or your internet connection is not sufficient.   
+**Ping** shows your network latency in milliseconds, the lower the better. Ping time contributes to overall delay (see below). The most probable causes for a high ping are that your distance to the server is too large.   
```

Comment:
Maybe: 

"The most probable cause~~s~~ for a high ping time ~~are~~ is that your distance to the server is too large."